lc-hax/Scripts/Patches/NoFallDamagePatch.cs

```diff
@@ -4,5 +4,5 @@
 
 [HarmonyPatch(typeof(PlayerControllerB), "DamagePlayer")]
 class NoFallDamagePatch {
-    static bool Prefix(CauseOfDeath causeOfDeath) => !Setting.DisableFallDamage || causeOfDeath != CauseOfDeath.Gravity;
+    static bool Prefix(CauseOfDeath causeOfDeath) => !(Setting.EnableGodMode || (Setting.DisableFallDamage && causeOfDeath == CauseOfDeath.Gravity));
```

Comment:
What’s up with this change? The previous way is a lot more concise and much easier to read.

Reply:
> What’s up with this change? Being in God mode naturally prevents taking fall damage.

just a test to figure whuch method needs to be blocked to make god mode not bug items, which it did not work, so ill revert it